Cargo.toml

```diff
@@ -31,8 +31,8 @@ debug = true
 ark-ff = { git = "https://github.com/arkworks-rs/algebra/" }
 ark-ec = { git = "https://github.com/arkworks-rs/algebra/" }
 ark-serialize = { git = "https://github.com/arkworks-rs/algebra/" }
-ark-crypto-primitives = { git = "https://github.com/arkworks-rs/crypto-primitives" }
```

Comment:
this will still have to change back once the rest of the deps are updated

Reply:
Of course! 👍🏻